### DISCUSSION TITLE
Windows Authentication in Blazor Server App don't work on IIS

### DISCUSSION THREAD

**Discussion by ashokoienia** [dotnet/aspnetcore#28086] [Q&A] [answered]
Hi All, I'm building a Blazor Server app, and have issue with Windows Authentication. The App works fine in IIS Express but after deploying on IIS authentication fails. Windows Authentication is installed on IIS and is activated for web site on IIS-Manager also, but the failure still. The user.Identity.IsAuthenticated is false and user.Identity.Name is null or empty. 
Now I'm a bit confused because as in following code snippet at least error message must be shown, but it is nothing to show:

`

    <AuthorizeView>
         @if (@getName(@context.User.Identity.Name) == false)
          {
                          <p>@strName haben Sie kein Zugriff!</p>
          }
          else
          {
                         <p>Hi, @strName !</p>
           }
     </AuthorizeView>

     @code {
             string strName { get; set; } = "Leider";

             bool getName(string sName)
             {
                 if (!string.IsNullOrWhiteSpace(sName))
                        strName = sName;
                 return !string.IsNullOrWhiteSpace(strName);
             }
     }

` 

Is anything that I missed probably? I could not found in anything in documrntations?
Any Idea? Thanks

**Comment by Tratcher** [comment]
Related: https://github.com/dotnet/aspnetcore/issues/27674

**Comment by ashokoienia** [comment, marked as answer]
@Tratcher  Thanks for your answer. Its over with this #27674 issue. I made mistake in setting IIS-Manager authentication settings.
Actually I did not deactivate "Anonymous Authentication" and it was my mistake.
Thanks a lot
